packages/custom_lint/test/cli_test.dart

```diff
@@ -258,27 +334,27 @@ class _Lint extends DartLintRule {
   ) {
     final line2 = resolver.lineInfo.getOffsetOfLine(1);
     reporter.reportErrorForOffset(
-      const LintCode(name: 'x2', problemMessage: 'x2'),
+      const LintCode(name: 'x2', problemMessage: 'x2', errorSeverity: ErrorSeverity.WARNING),
```

Comment:
Why these changes?

Reply:
The reason is the same as [here](https://github.com/invertase/dart_custom_lint/pull/129#discussion_r1233210398).